source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -326,10 +326,23 @@ void CacheFilter::getBody() {
   // posted callback.
   CacheFilterWeakPtr self = weak_from_this();
 
+  // We don't want to request more than a buffer-size at a time from the cache.
+  uint64_t fetch_size_limit = encoder_callbacks_->encoderBufferLimit();
+  // If there is no buffer size limit, we probably still want *some* constraint,
+  // because reading a gigabyte of cache entry in one go is always going to cause
+  // some kind of problem. Everyone knows 64MB should be enough for anyone.
+  if (fetch_size_limit == 0) {
+    fetch_size_limit = 64 * 1024 * 1024;
+  }
+  AdjustedByteRange fetch_range = {remaining_ranges_[0].begin(),
```

Comment:
I feel like a temp for remaining_ranges_[0] might help readability and maybe speed, not that speed matters for this that much.

Reply:
I couldn't think of a good name for such a temporary that would make it more readable rather than less, and `[0]` is definitely compiler-optimized away to be identical performance-wise. (I'm 99.9% confident of that, and only 99.8% confident that making a temporary local reference is also optimized away, haha.)

Reply:
i'd suggest `first_range`or `range0` as a name?